src/sth_app.js

```diff
@@ -24,7 +24,9 @@
     }
 
     if (err) {
-      sthLogger.error(err);
+      sthLogger.fatal(err.toString(), {
+        operationType: sthConfig.OPERATION_TYPE.SHUTDOWN
```

Comment:
Yeap, absolutely right! The winston (Node.js) module allows passing a "metadata" (object) with additional information. I used this "metadata" to pass to the logger the context in which the reason of the logging took place, according to the FIWARE logging format ;)